R/plot_cbass.R

```diff
@@ -1016,10 +1034,12 @@ cbass_dendro_plotly <- function(x,
       allTXT_dynamic <- rbind(allTXT_dynamic, allTXT_per)
     }
 
+    count <- NULL # for CMD check
```

Comment:
This trick isn't great - if this is for use in a "tidyverse" function, it's better to change things to use the `.data` pronoun. 

This code could become

```
c <- dynamic_seg %>% group_by(.data$x, .data$xend) %>% summarize(count = n())
dynamic_sec <- arrange(merge(dynamic_seg, c), .data$Regularization, desc(.data$count), .data$x, .data$y)
```

I think you can also make this a bit cleaner with the `dplyr::count` function.

Reply:
In general, the global variables thing isn't great because it's just "tricking" the check while the `.data` thing will actually make sure there's a real variable by that name. If you set `x <- NULL` and then try `select(data, x)` on `data` without an `x` column, you get a really nasty error.